modules/telco-ran-414-whats-new-ref-design.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * telco_ref_design_specs/ran/telco-ran-ref-design-spec.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="telco-ran-whats-new-ref-design_{context}""]
+= What's new in {product-title} {product-version} for telco RAN
```

Comment:
There is already a "whats new in OpenShift" delivered by PM. I don't want this to supplant that. Perhaps we can rephrase this along the lines of:
```suggestion
= Impact of new {product-title} {product-version} features in telco RAN reference configuration
```